doc/source/user/numpy-for-matlab-users.rst

```diff
@@ -7,220 +7,125 @@ NumPy for MATLAB users
-       (though for matrix multiplication, one can use the ``@`` operator
-       in python 3.5 and above).
-
-   * - MATLAB uses 1 (one) based indexing. The initial element of a
-       sequence is found using a(1).
+   :class: docutils
+
+   * - In MATLAB, the basic type, even for scalars, is a
+       multidimensional array. Array assignments in MATLAB are stored as
+       2D arrays of double precision floating point numbers, unless you
+       specify the number of dimensions and type.  Operations on the 2D
+       instances of these arrays are modeled on matrix operations in
+       linear algebra. 
+
+     - In NumPy, the basic type is a multidimensional ``array``.  Array
+       assignments in NumPy are usually stored as n-dimensional arrays with the
+       minimum type required to hold the objects in sequence, unless you
+       specify the number of dimensions and type. NumPy performs
+       operations element-by-element, so multiplying 2D arrays with
+       ``*`` is not a matrix multiplication -- it's an
+       element-by-element multiplication. (The ``@`` operator, available
+       since Python 3.5, can be used for conventional matrix
+       multiplication.)
+
+   * - MATLAB numbers indices from 1; ``a(1)`` is the first element.
        :ref:`See note INDEXING <numpy-for-matlab-users.notes>`
-     - Python uses 0 (zero) based indexing. The initial element of a
-       sequence is found using a[0].
-
-   * - MATLAB's scripting language was created for doing linear algebra.
-       The syntax for basic matrix operations is nice and clean, but the API
-       for adding GUIs and making full-fledged applications is more or less
-       an afterthought.
-     - NumPy is  based on Python, which was designed from the outset to be
-       an excellent general-purpose programming language.  While MATLAB's
+     - NumPy, like Python, numbers indices from 0; ``a[0]`` is the first
+       element.
+
+   * - MATLAB's scripting language was created for linear algebra so the
        syntax for some array manipulations is more compact than
-       NumPy's, NumPy (by virtue of being an add-on to Python) can do many
-       things that MATLAB just cannot, for instance dealing properly with
-       stacks of matrices.
+       NumPy's. On the other hand, the API for adding GUIs and creating 
+       full-fledged applications is more or less an afterthought.
+     - NumPy is  based on Python, a
+       general-purpose language.  The advantage to NumPy
+       is access to Python libraries including: `SciPy
+       <https://www.scipy.org/>`_, `Matplotlib <https://matplotlib.org/>`_,
+       `Pandas <https://pandas.pydata.org/>`_, `OpenCV <https://opencv.org/>`_,
+       and more. In addition to Python-specific packages, much `scientific
+       computing software
+       <https://en.wikipedia.org/wiki/List_of_Python_software#Embedded_as_a_scripting_language>`_
+       use Python as a scripting language. 
```

Comment:
Looking at that list, not a particularly large fraction of the applications strike me as "scientific computing". I'd be inclined to either throw out this sentence, or phrase it as:
```suggestion
       and more. In addition, Python is often `embedded as a scripting language
       <https://en.wikipedia.org/wiki/List_of_Python_software#Embedded_as_a_scripting_language>`_
       in other software, allowing NumPy to be used there too. 
```